sourcetools/objectmodel/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+################################################################################
+# Copyright (c) 2017, 2018 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+################################################################################
+add_jar( objectmodel
+	com/ibm/j9tools/om/AsmBuilder.java
+	com/ibm/j9tools/om/BuildInfo.java
+	com/ibm/j9tools/om/BuildSpec.java
+	com/ibm/j9tools/om/ConfigDirectory.java
+	com/ibm/j9tools/om/DefaultSizes.java
+	com/ibm/j9tools/om/FeatureDefinition.java
+	com/ibm/j9tools/om/Feature.java
+	com/ibm/j9tools/om/FlagDefinition.java
+	com/ibm/j9tools/om/FlagDefinitions.java
+	com/ibm/j9tools/om/Flag.java
+	com/ibm/j9tools/om/FlagVerifier.java
+	com/ibm/j9tools/om/IFlagContainer.java
+	com/ibm/j9tools/om/InvalidAsmBuilderException.java
+	com/ibm/j9tools/om/InvalidBuildInfoException.java
+	com/ibm/j9tools/om/InvalidBuildSpecException.java
+	com/ibm/j9tools/om/InvalidConfigurationException.java
+	com/ibm/j9tools/om/InvalidDefaultSizesException.java
+	com/ibm/j9tools/om/InvalidFactoryException.java
+	com/ibm/j9tools/om/InvalidFeatureDefinitionException.java
+	com/ibm/j9tools/om/InvalidFeatureException.java
+	com/ibm/j9tools/om/InvalidFlagDefinitionsException.java
+	com/ibm/j9tools/om/InvalidFlagException.java
+	com/ibm/j9tools/om/InvalidJCLException.java
+	com/ibm/j9tools/om/InvalidSourceException.java
+	com/ibm/j9tools/om/InvalidSpecIDException.java
+	com/ibm/j9tools/om/io/AbstractParser.java
```

Comment:
Can you reorder this so all classes in a package are kept together?

How does the `schema` directory get included in the resulting jar?